### PR TITLE
Update capacitypolicy.md

### PR DESCRIPTION
**Pull request by tomersh26**
#assign-reviewer:YaelSchuster

### CODE REVIEW

**Comment by tomersh26** [comment]
#assign-reviewer:yifatsor

**Comment by tomersh26** [comment]
#assign-reviewer:shsagir

**Comment by tomersh26** [comment]
#assign-reviewer:yaelschuster

**Comment by Court72** [comment]
@shsagir

Can you review the proposed changes? 

When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: data-explorer/kusto/management/capacitypolicy.md [hunk 1]
```diff
@@ -99,6 +99,7 @@ The policy can be used to change concurrency settings for [materialized views](m
 
 The effective value for `concurrent operations` is automatically adjusted by the system in the range
 [`ClusterMinimumConcurrentOperations`,`ClusterMaximumConcurrentOperations`], based on the number of materialized views in the cluster and the cluster's CPU.
+Use [`.show capacity`](../management/diagnostics.md#show-capacity) materialized-view to show the effective value of `concurrent operations`.
```
Comment (v-dkorn) [inline data-explorer/kusto/management/capacitypolicy.md:102]:
```suggestion

Run the [`.show capacity`](../management/diagnostics.md#show-capacity) command to show the effective value of `concurrent operations`.
```

**Comment by v-dkorn** [comment]
I'm speaking with Tomer to review this PR.

**Comment by v-dkorn** [comment]
#please-close 

I opened a separate PR to introduce these changes. Thanks Tomer!

**Pull request closed without merging**